python/src/block/DelayedNeutronPrecursorData.python.cpp

```diff
@@ -33,33 +33,42 @@ void wrapDelayedNeutronPrecursorData( python::module& module, python::module& )
   block
   .def(
 
-    python::init< double, std::vector< double >, std::vector< double > >(),
-    python::arg( "constant" ), python::arg( "energies" ),
-    python::arg( "probabilities" ),
+    python::init< unsigned int, double,
+                  std::vector< double >, std::vector< double > >(),
+    python::arg( "number" ), python::arg( "constant" ),
+    python::arg( "energies" ), python::arg( "probabilities" ),
     "Initialise the block assuming linear interpolation\n\n"
     "Arguments:\n"
     "    self             the block\n"
+    "    number           the precursor group number\n"
```

Comment:
Is it expected that groups will be sparse or stored out-of-order?

Reply:
I was planning on sorting the groups in order but for some reason I decided not to. I forgot why.